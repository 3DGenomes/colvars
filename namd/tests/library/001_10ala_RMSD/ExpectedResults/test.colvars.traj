# step         rmsdA                 fs_rmsdA               rmsdB                 fs_rmsdB               rmsdC                 fs_rmsdC               rmsdD                 fs_rmsdD               rmsdE                 fs_rmsdE             
           0    2.61009345658189e-01  0.00000000000000e+00   1.27145912684355e+01  0.00000000000000e+00   6.68502465870546e-01  0.00000000000000e+00   9.31703240469482e-01  0.00000000000000e+00   1.24224459145532e+00  0.00000000000000e+00
           1    2.58818393903831e-01 -1.91887408131368e-02   1.27140121208706e+01 -1.32011132830181e-02   6.73190616747935e-01  5.01068124575828e-02   9.49254457200972e-01  1.42994471091580e-01   1.24048463409653e+00  8.71318461837457e-02
           2    2.56438350855573e-01  2.55367153946768e+00   1.27128708523953e+01  9.38788276131300e+00   6.78180296003691e-01 -1.08435949513821e+00   9.64812723825183e-01  5.88650780899527e+00   1.23298183505426e+00  5.72592174280893e+00
           3    2.55470552845568e-01  2.00163123497541e+00   1.27111617849414e+01  2.18841273532013e+01   6.83326275548580e-01  2.62402062027840e+00   9.81216988715104e-01  1.02131671998838e+01   1.22048081971971e+00  1.35865169142587e+01
           4    2.57254812251994e-01 -1.77743638649465e+01   1.27091572081105e+01  3.61766442135220e+01   6.89831429569709e-01  4.15859282176564e+00   9.98303940524606e-01  8.31804097470430e+00   1.20402305611622e+00  1.90768219573323e+01
           5    2.64206645896857e-01 -7.07784069086467e+01   1.27069627981967e+01  5.03331481462640e+01   6.98662653718267e-01 -4.64960324053547e+00   1.01558861199617e+00 -4.14075181588912e+00   1.18566775936962e+00  1.64559693841100e+01
           6    2.77743825801463e-01 -1.57473162645268e+02   1.27047223136543e+01  6.37778719967148e+01   7.10223227873656e-01 -2.89444999710756e+01   1.03449018550297e+00 -2.86353377247179e+01   1.16657824072359e+00  1.77033830534923e+00
           7    2.98630264777304e-01 -2.65679525376305e+02   1.27026329528893e+01  7.75490426327508e+01   7.25698630961578e-01 -7.06398945273838e+01   1.05261679481539e+00 -6.29357148664749e+01   1.14931188609770e+00 -2.61785689238170e+01
           8    3.26772312127377e-01 -3.78290213906659e+02   1.27008539352054e+01  9.30411547139643e+01   7.44858567008818e-01 -1.29635758948345e+02   1.07241693768652e+00 -1.04437900552978e+02   1.13493293678682e+00 -6.91496608561897e+01
           9    3.62109050292986e-01 -4.86557439743461e+02   1.26994228476460e+01  1.16743244271756e+02   7.67990578429051e-01 -2.08060386260715e+02   1.09311214875741e+00 -1.50565487273726e+02   1.12491435503097e+00 -1.29418075517884e+02
          10    4.03214372791890e-01 -6.06945498854327e+02   1.26985826095870e+01  1.66379461913882e+02   7.94288130577766e-01 -3.20193175483313e+02   1.12095970604269e+00 -2.02994103749918e+02   1.12086493086102e+00 -2.18542567741985e+02
          11    4.47969625891838e-01 -7.88521557174713e+02   1.26986342267581e+01  2.85306539248978e+02   8.22066191796782e-01 -5.04919760778955e+02   1.16487123045747e+00 -2.70201445043731e+02   1.12298477968616e+00 -3.71580606186805e+02
          12    4.93416854558069e-01 -1.09563754808584e+03   1.27002236913477e+01  5.62471743663986e+02   8.47932206839753e-01 -8.21481079420292e+02   1.23014495873972e+00 -3.57183536249443e+02   1.12968516938995e+00 -6.51955502331831e+02
          13    5.35116333798935e-01 -1.45826282502162e+03   1.27044324973302e+01  1.00294745190441e+03   8.66939557805021e-01 -1.19758518320606e+03   1.33073535261868e+00 -4.06652052966841e+02   1.13696774493663e+00 -1.02783265538364e+03
          14    5.73902331567880e-01 -1.51910646462223e+03   1.27113385970065e+01  1.08472947574085e+03   8.80544202356082e-01 -1.22124200693915e+03   1.46574492585183e+00 -3.64102814206821e+02   1.14543683145157e+00 -1.10933156597762e+03
          15    6.14139233331071e-01 -1.43622374026783e+03   1.27197410945860e+01  6.84770492171827e+02   8.94515016428242e-01 -1.01426204246443e+03   1.62316181669400e+00 -3.85449356263034e+02   1.15965456578410e+00 -9.18924711312455e+02
          16    6.56322604809923e-01 -1.46248547166690e+03   1.27284948552794e+01  3.61351419899097e+02   9.11769754689333e-01 -9.37880678655358e+02   1.79505152631988e+00 -4.53958274633546e+02   1.18272728723987e+00 -8.16224506355988e+02
          17    6.98500007016764e-01 -1.50563289057680e+03   1.27367099235725e+01  1.84309558238818e+02   9.31733151186190e-01 -9.56177532518011e+02   1.98732735495308e+00 -4.88044250835391e+02   1.21467228400512e+00 -8.03076061836977e+02
          18    7.38801505436272e-01 -1.52234972829864e+03   1.27437884051266e+01  7.97607928180288e+01   9.53529754685968e-01 -1.00708268508978e+03   2.20735144936535e+00 -4.81931051957053e+02   1.25490915594631e+00 -8.24820653584312e+02
          19    7.75631489442574e-01 -1.51339936555352e+03   1.27492039260640e+01  1.58211610603196e+01   9.76171125578086e-01 -1.05691123139395e+03   2.47344509252077e+00 -4.43963780587357e+02   1.30283754506013e+00 -8.49364378255247e+02
          20    8.07503719583766e-01 -1.47268241448263e+03   1.27525892413933e+01 -1.63162137787955e+01   9.98716781705996e-01 -1.08074862057924e+03   2.79853882441752e+00 -3.81467052372561e+02   1.35754144284582e+00 -8.57079075371780e+02
