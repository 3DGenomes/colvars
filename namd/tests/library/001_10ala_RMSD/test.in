colvarsTrajFrequency 1


colvar {
    name rmsdA

    outputSystemForce on
    #  outputAppliedForce on
    width 1.0 

    lowerBoundary 0.0
    upperBoundary 5.0

    lowerWallConstant 100.0
    upperWallConstant 100.0

    rmsd {
        atoms {
            atomNumbers { 1 2 3 4 5 6 7 8 }
        }
        refpositionsfile ../Common/da.xyz
    }
} 


colvar {
    name rmsdB

    outputSystemForce on

    width 1.0 

    lowerBoundary 0.0
    upperBoundary 5.0

    lowerWallConstant 100.0
    upperWallConstant 100.0

    rmsd {
        atoms {
            atomNumbers { 1 2 3 4 5 6 7 8 }
            rotateReference
            centerReference
            refpositionsfile ../Common/pi-ideal.xyz
        }
        refpositionsfile ../Common/da.xyz
    }
} 


colvar {
    name rmsdC

    outputSystemForce on

    width 1.0 

    lowerBoundary 0.0
    upperBoundary 5.0

    lowerWallConstant 100.0
    upperWallConstant 100.0

    rmsd {
        atoms {
            atomNumbers { 1 2 3 4 5 6 7 8 }
            centerReference
            refpositionsfile ../Common/da.xyz
        }
        refpositionsfile ../Common/da.xyz
    }
} 


colvar {
    name rmsdD

    outputSystemForce on

    width 1.0 

    lowerBoundary 0.0
    upperBoundary 5.0

    lowerWallConstant 100.0
    upperWallConstant 100.0

    rmsd {
        atoms {
            atomNumbers { 1 2 3 4 5 6 7 8 }
            rotateReference
            refpositionsfile ../Common/da.xyz
        }
        refpositionsfile ../Common/da.xyz
    }
} 


colvar {
    name rmsdE

    outputSystemForce on

    width 1.0 

    lowerBoundary 0.0
    upperBoundary 5.0

    lowerWallConstant 100.0
    upperWallConstant 100.0

    rmsd {
        atoms {
            atomNumbers { 1 2 3 4 5 6 7 8 }
            centerReference no
            rotateReference no
        }
        refpositionsfile ../Common/da.xyz
    }
} 


harmonic {
    colvars        rmsdA rmsdB rmsdC rmsdD rmsdE
    forceConstant  1.0
    centers        0.0   0.0   0.0   0.0   0.0   
}
