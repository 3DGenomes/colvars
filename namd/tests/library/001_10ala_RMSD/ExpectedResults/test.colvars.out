colvars: ----------------------------------------------------------------------
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # indexFile = "" [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 2000
colvars: # colvarsTrajAppend = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = rmsdA
colvars:   Initializing a new "rmsd" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # disableForces = off [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "atoms" defined, 8 atoms initialized: total mass = 43.069.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = ../Common/da.pdb
colvars:     # refPositionsCol = "" [default]
colvars:     Enabling "centerReference" and "rotateReference", to minimize RMSD before calculating it as a variable: if this is not the desired behavior, disable them explicitly within the "atoms" block.
colvars:   All components initialized.
colvars:   # width = 1
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 100
colvars:   # lowerWall = 0 [default]
colvars:   # upperBoundary = 5
colvars:   # upperWallConstant = 100
colvars:   # upperWall = 5 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = rmsdB
colvars:   Initializing a new "rmsd" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # disableForces = off [default]
colvars:       # centerReference = on
colvars:       # rotateReference = on
colvars:       # enableFitGradients = on [default]
colvars:       # refPositions =  [default]
colvars:       # refPositionsFile = ../Common/pi-ideal.pdb
colvars:       # refPositionsCol = "" [default]
colvars:       Warning: atom group "atoms" will be aligned to a fixed orientation given by the reference positions provided.  If the internal structure of the group changes too much (i.e. its RMSD is comparable to its radius of gyration), the optimal rotation and its gradients may become discontinuous.  If that happens, use refPositionsGroup (or a different definition for it if already defined) to align the coordinates.
colvars:       Atom group "atoms" defined, 8 atoms initialized: total mass = 43.069.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = ../Common/da.pdb
colvars:     # refPositionsCol = "" [default]
colvars:     WARNING: explicit fitting parameters were provided for atom group "atoms".
colvars:   All components initialized.
colvars:   # width = 1
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 100
colvars:   # lowerWall = 0 [default]
colvars:   # upperBoundary = 5
colvars:   # upperWallConstant = 100
colvars:   # upperWall = 5 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = rmsdC
colvars:   Initializing a new "rmsd" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # disableForces = off [default]
colvars:       # centerReference = on
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       # refPositions =  [default]
colvars:       # refPositionsFile = ../Common/da.pdb
colvars:       # refPositionsCol = "" [default]
colvars:       Atom group "atoms" defined, 8 atoms initialized: total mass = 43.069.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = ../Common/da.pdb
colvars:     # refPositionsCol = "" [default]
colvars:     WARNING: explicit fitting parameters were provided for atom group "atoms".
colvars:   All components initialized.
colvars:   # width = 1
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 100
colvars:   # lowerWall = 0 [default]
colvars:   # upperBoundary = 5
colvars:   # upperWallConstant = 100
colvars:   # upperWall = 5 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = rmsdD
colvars:   Initializing a new "rmsd" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # disableForces = off [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = on
colvars:       # enableFitGradients = on [default]
colvars:       # refPositions =  [default]
colvars:       # refPositionsFile = ../Common/da.pdb
colvars:       # refPositionsCol = "" [default]
colvars:       Warning: atom group "atoms" will be aligned to a fixed orientation given by the reference positions provided.  If the internal structure of the group changes too much (i.e. its RMSD is comparable to its radius of gyration), the optimal rotation and its gradients may become discontinuous.  If that happens, use refPositionsGroup (or a different definition for it if already defined) to align the coordinates.
colvars:       Atom group "atoms" defined, 8 atoms initialized: total mass = 43.069.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = ../Common/da.pdb
colvars:     # refPositionsCol = "" [default]
colvars:     WARNING: explicit fitting parameters were provided for atom group "atoms".
colvars:   All components initialized.
colvars:   # width = 1
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 100
colvars:   # lowerWall = 0 [default]
colvars:   # upperBoundary = 5
colvars:   # upperWallConstant = 100
colvars:   # upperWall = 5 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = rmsdE
colvars:   Initializing a new "rmsd" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # disableForces = off [default]
colvars:       # centerReference = off
colvars:       # rotateReference = off
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "atoms" defined, 8 atoms initialized: total mass = 43.069.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = ../Common/da.pdb
colvars:     # refPositionsCol = "" [default]
colvars:     WARNING: explicit fitting parameters were provided for atom group "atoms".
colvars:   All components initialized.
colvars:   # width = 1
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 100
colvars:   # lowerWall = 0 [default]
colvars:   # upperBoundary = 5
colvars:   # upperWallConstant = 100
colvars:   # upperWall = 5 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 5 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { rmsdA, rmsdB, rmsdC, rmsdD, rmsdE }
colvars:   # outputEnergy = off [default]
colvars:   # forceConstant = 1
colvars:   # centers = { 0, 0, 0, 0, 0 }
colvars:   # targetCenters = { 0, 0, 0, 0, 0 } [default]
colvars:   # targetForceConstant = 0 [default]
colvars:   # outputCenters = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: The final output state file will be "test.colvars.state".
colvars: Writing to colvar trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.colvars.state".
